Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli

```diff
@@ -89,6 +89,9 @@ void Surface::ApplySpecularAA()
     float kernelRoughnessA2 = min(2.0 * variance , varianceThresh );
     float filteredRoughnessA2 = saturate ( roughnessA2 + kernelRoughnessA2 );
     roughnessA2 = filteredRoughnessA2;
+
+    roughnessA = sqrt(roughnessA2);
```

Comment:
Why is there CalculateSpecularAA and ApplySpecularAA both doing the same things but now we are modifying one? It all seems very confusing. Can we just remove one entirely and refactor it out a bit?

Reply:
ApplySpecularAA exists in various surface classes before CalculateSpecularAA is created. But some have linear roughness set back, some don't, like this one. Now all the shaders except multilayer PBR in Feature\Common\Assets\Shaders\Materials have been refactored to use CalculateRoughnessValues, it calls CalculateSpecularAA and sets back roughness value.
I am not sure why multilayer PBR is left untouched. If we are doing another refactoring, I think it should go into another PR.